quickstarts/Search_Grounding.ipynb

```diff
@@ -0,0 +1,421 @@
+{
```

Comment:
<p>If the package is out when you merge, can you update this?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google-gemini/cookbook/pull/314/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>Done.</p>